lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * View for an individual team.
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', 'underscore', 'gettext', 'teams/js/views/team_discussion',
```

Comment:
@symbolist have you taken a look at this code? It will need to sync up with the Team page you are creating.